static/docs/get-started/metrics.md

```diff
@@ -27,7 +27,7 @@ Let's again commit and save results:
 
 ```dvc
 $ git add evaluate.dvc auc.metric
-$ git commit -m "add evaluation step to the pipeline"
```

Comment:
in this case it's a specific pipeline that we have?

Reply:
Sure, I was just trying to avoid "THE pipeline" as much as possible in the docs. I think it can cause the reader to think DVC can only handle one pipeline per project. But I  can revert these changes in the commit messages if you prefer. Lmk please

Reply:
I would revert those. Don't think it's worth optimizing it so aggressively. If people understand the pipelines functionality they most likely can figure out that it's not a single pipeline.

Reply:
OK! Will revert in next PR.